### PR TITLE
ESS Auth Description Update

### PR DESCRIPTION
**Pull request by haydenlam**

### CODE REVIEW

File: src/api-reference/common/ess/subscription-management.markdown [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+* [Browse existing subscriptions](#get-subscription-list)
+* [Delete your subscription](#delete-subscription)
+* [Request Example](#example)
+
+## <a name="ac-jwt"></a> 1. Access Control (JWT)
+The Event Subscription Service (ESS) APIs uses the standard Concur OAuth2 Framework for Authentication and Authorization via JSON Web Token (JWT).
+
+The authentication is done by using a JWT which represents the event subscriber. This can be obtained by calling the <a href="https://developer.concur.com/api-reference/authentication/apidoc.html#client_credentials">Client Credentials</a> grant type using your OAuth2 credentials. The JWT generated by this grant will have a `JWT Type` of `App` where your `Application ID` is the `Subject` of the Token. 
+
+For Authrization, the Application needs to contain the scope `events.topic.read` in order to access the ESS APIs. This scope will be inherited by the JWT during the Client Credentials Grant which is then checked by the ESS API.
```
Comment (RoAnnC1) [inline src/api-reference/common/ess/subscription-management.markdown]: Authorization is misspelled.

File: src/api-reference/common/ess/subscription-management.markdown [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+
+Pass the JWT in the Post Header when making calls to the ESS API:
+```
+Authorization: Bearer <JWT>
+```
+The Client Credentials JWT has a lifetime of 60 minutes and can only be obtained again via the Client Credentials Grant.
+
+## <a name="get-topics"></a> 2. Browse available topics
+Before you create any subscription you need to verify that you have sufficient access to the topic. If that request returns empty you need to get in touch with your assigned contact from Concur to set proper scopes to you Concur Application Records (how do I call that and not confuse them with their endpoint/application?).
```
Comment (RoAnnC1) [inline src/api-reference/common/ess/subscription-management.markdown]: SAP Concur
scopes to your

File: src/api-reference/common/ess/subscription-management.markdown [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+```
+**Response**
+```json
+["public.test"]
+```
+
+## <a name="put-subscription"></a> 3. Create a subscription
+To create a subscription you need to
+1. know and have sufficient access to the **topic**
+1. get your receiving **endpoint** up running, <a href="https://developer.concur.com/api-reference/common/ess/getting-started.html#endpoint-requirements">endpoint
```
Comment (RoAnnC1) [inline src/api-reference/common/ess/subscription-management.markdown]: Delete "up"

File: src/api-reference/common/ess/subscription-management.markdown [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+requirements</a>     
+1. come up with unique name (id) of your subscription. Subscription names are globally unique, and you will get error if name is already occupied.
```
Comment (RoAnnC1) [inline src/api-reference/common/ess/subscription-management.markdown]: "an error if the name"

**Comment by haydenlam** [comment]
Corrections made.

**Comment by RoAnnC1** [comment]
:shipit:

**Comment by usandre** [comment]
When is it going to be merged?

**Comment by usandre** [comment]
@jonikamelcher why I don't see this on the portal?

**Comment by jonikamelcher** [comment]
@usandre Are you looking at the preview site? https://preview.developer.concur.com/api-reference/common/ess/subscription-management.html. This will be moved to the livesite after we merge some additional, unrelated changes.

**Pull request merged**